tools/stern/src/Main.hs

```diff
@@ -8,4 +8,4 @@ main :: IO ()
 main = getOptions desc Nothing defaultPath >>= start
   where
     desc = "Stern - Backoffice Service"
-    defaultPath = "/etc/wire/stern/conf/stern.yaml"
+    defaultPath = "./stern.example.yaml"
```

Comment:
please undo this change, as it breaks the defaults on kubernetes-deployed environments, and it's inconsistent with regards to the other services.

Reply:
oops  :-|=|  that wasn't supposed to go in.  sorry!